### PR TITLE
jgrss/dependencies

### PR DESCRIPTION
**Pull request by jgrss**
## What is this PR changing?
* Moved deprecated `xarray.open_rasterio` to this project
  * See `geowombat.backends.xarray_rasterio_()`

### CODE REVIEW

**Comment by jgrss** [comment]
@mmann1123 The main change here is the addition of `geowombat.backends.xarray_rasterio_` and, specifically, `open_rasterio()` (which is deprecated in Xarray).

**Comment by mmann1123** [comment]
> @mmann1123 The main change here is the addition of `geowombat.backends.xarray_rasterio_` and, specifically, `open_rasterio()` (which is deprecated in Xarray).

I was wondering about that

File: .circleci/config.yml [hunk 1]
```diff
@@ -55,26 +55,24 @@ jobs:
...
-            # This shouldn't be necessary with pyproject.toml
-            pip install Cython numpy
+            # This shouldn't be necessary with pyproject.toml but we need numpy for GDAL
+            pip install -U numpy
             pip install GDAL==$GDAL_VERSION
             # Install GeoWombat
-            pip install .
-            # We should be able to install these as extras (i.e., .[ml])
-            pip install sklearn-xarray rtree
+            pip install .[perf,ml]
```
Comment (mmann1123) [inline .circleci/config.yml:62]: We need to remember to update the readme about these changes

**Review by mmann1123 — APPROVED** [review]
Looks good. We need to update the read me to reflect installing ['ml'] etc

File: .circleci/config.yml (hunk 1 above)
Comment (jgrss) [reply .circleci/config.yml:62]: Good call

**Pull request merged**